rexml.gemspec

```diff
@@ -17,8 +17,6 @@ Gem::Specification.new do |spec|
   spec.license       = "BSD-2-Clause"
 
   spec.files         = [
-    ".gitignore",
-    ".travis.yml",
     "Gemfile",
```

Comment:
Could you also remove this and `Rakefile`?

Reply:
Indeed, it is useless too.

Reply:
```suggestion
```